code/game/gamemodes/cult/cult.dm

```diff
@@ -200,6 +201,7 @@
 /datum/game_mode/proc/remove_cultist(datum/mind/cult_mind, show_message = 1)
 	if(cult_mind in cult)
 		cult -= cult_mind
+		cult_mind.current.faction -= "cult"

```

Comment:
This does not remove the cult faction from the mob being deconverted, for clarity. I'm confused as to why it doesn't work.